phpcs.xml

```diff
@@ -9,13 +9,15 @@
   <exclude-pattern>*/node_modules/*</exclude-pattern>
   <exclude-pattern>*/vendor/*</exclude-pattern>
 
-  <!-- Yoda conditions, we must ignore -->
+  <rule ref="Generic.Files.LowercasedFilename.NotFound">
```

Comment:
Put this next to the other `WordPress.File` rule override.

Reply:
The rules are grouped by "namespace"

Reply:
Yes they are, I read wrong, the other is `Generic` no `WordPress`, my bad.